nft-book-mint-and-list/iscn.js

```diff
@@ -20,8 +21,12 @@ let iscnSigningClient;
 let signingStargateClient;
 
 async function getWallet() {
+  if (!MNEMONIC && !PRIVATE_KEY) throw new Error('Need MNEMONIC or PRIVATE_KEY');
+  if (MNEMONIC && PRIVATE_KEY) console.warn('Both MNEMONIC and PRIVATE_KEY are provided, using MNEMONIC');
   if (!signingWallet) {
-    signingWallet = await DirectSecp256k1HdWallet.fromMnemonic(MNEMONIC, { prefix: 'like' });
+    signingWallet = MNEMONIC
+      ? await DirectSecp256k1HdWallet.fromMnemonic(MNEMONIC, { prefix: 'like' })
+      : await DirectSecp256k1Wallet.fromKey(PRIVATE_KEY, { prefix: 'like' });
```

Comment:
I think `DirectSecp256k1Wallet.fromKey` accept prefix as string param, i.e. `await DirectSecp256k1Wallet.fromKey(PRIVATE_KEY, 'like');`
Please verify if thats the case

Reply:
Indeed a wrong function signature, fixed